docs/guides/08-debugging/application.md

```diff
@@ -40,19 +40,59 @@ error: aborting due to previous error
 For more information about this error, try `rustc --explain E0425`.
 ```
 
-## WebView JS Console
+## WebView Console
 
 Right-click in the WebView, and choose `Inspect Element`. This opens up a web-inspector similar to the Chrome or Firefox dev tools you are used to.
```

Comment:
one screenshot for each OS? with tabs?

Reply:
Could possibly do that, I can grab one of macOS and put it here (and can put the tabs together too)

Reply:
<img width="204" alt="Screenshot 2022-05-30 at 09 53 16" src="https://user-images.githubusercontent.com/15347255/170955646-4a263268-99b1-4c7b-9f01-27835aed698a.png">

Something like this? It could also be that this is overkill looking at it now ;)

Reply:
seems like i'm too dumb to make several screenshots the same size and menu position :D

Reply:
Honestly, don't worry about it, this specific place probably doesn't make sense. And always easy to add them in later if we decide we want to later.

Reply:
ok PR done then

Reply:
yeah i'll add that